Created a new chatbot !!!

SrijaVuppala295 opened the pull request:
![image](https://github.com/user-attachments/assets/8533edaf-84a8-487e-9153-e11859f1610c)

# Fixes : #434

samyakmaitre commented:
Thank you for your contribution! The merge has been completed successfully, and the labels have been added. I appreciate your efforts, and don't forget to connect with me on GitHub!

Best regards,
Samyak

SrijaVuppala295 commented:
@samyakmaitre , please increase the level from level2 to level3 !! I have added all the related stuff in the chatbot and changed according to the theme !!!
